packages/browser-extension/source/util/localizations.ts

```diff
@@ -0,0 +1,244 @@
+          tr.appendChild(td)
+      })
+      tableBody?.querySelectorAll('tr').forEach((tr, index) => {
+        const input = panelWindow.document.createElement('input')
+        const td = panelWindow.document.createElement('td')
+
+        input.value = slovak[index]
+        td.appendChild(input)
+          tr.append(td);
+        })   
+  
+}
+
+export function saveTableValuesAndParseBack(tableBody:any, positions:any[]) {
+  let englishTable:string[] = [] 
+  let slovakTable:string[] = []
+  for (var r = 0, n = tableBody.rows.length; r < n; r++) {
+      for (var c = 0, m = tableBody.rows[r].cells.length; c < m; c++) {
+
+        if (tableBody.rows[r].cells[c].childNodes[0].nodeType == Node.TEXT_NODE ) {
+          englishTable = [...englishTable, tableBody.rows[r].cells[c].innerHTML]
+        } else {
+          slovakTable = [...slovakTable, tableBody.rows[r].cells[c].getElementsByTagName('input')[0]?.value]
+          positions[r].text = tableBody.rows[r].cells[c].getElementsByTagName('input')[0]?.value
+        }
+      }
+  }
+  return {englishTable, slovakTable, positions}
+    
+}
+
+export function saveTableToObjects(englishTable:string[], slovakTable:string[]) {
+  let table:any = [] 
+  englishTable.forEach((english:string)=>{
+    table = [...table, {en:english, sk:''}]
+  })
+  slovakTable.forEach((slovak:string, index)=>{
+    table[index].sk = slovak 
+  })
+  return table
+}
+
+export function createObjectLiteralFromTable(locales:[]) {
+  let tsArray:any[] = []
+  factory.createExpressionStatement(factory.createObjectLiteralExpression(
+    [
+      factory.createPropertyAssignment(
+        factory.createIdentifier("export"),
+        factory.createIdentifier("")
+      ),
+      factory.createPropertyAssignment(
+        factory.createIdentifier("default"),
+        factory.createObjectLiteralExpression(
+          tsArray,
+          true
+        )
+      )
+    ],
+    false
+  ))
+
+  locales.forEach((locale:any)=>{
+    console.log(factory.createPropertyAssignment(
+      factory.createStringLiteral(locale.en),
+      factory.createStringLiteral(locale.sk)
+    ))
+    tsArray = [...tsArray,factory.createPropertyAssignment(
+      factory.createStringLiteral(locale.en),
+      factory.createStringLiteral(locale.sk)
+    ) ]
+  })
+
+  console.log("TS array", tsArray)
+  
+  return [
+    factory.createExpressionStatement(factory.createObjectLiteralExpression(
+      [
+        factory.createPropertyAssignment(
+          factory.createIdentifier("export"),
+          factory.createIdentifier("")
+        ),
+        factory.createPropertyAssignment(
+          factory.createIdentifier("default"),
+          factory.createObjectLiteralExpression(
+            tsArray,
+            true
+          )
+        )
+      ],
+      false
+    ))
+  ];
+}
+
+export function createFileFromAST(ast:any){
+  const printer = ts.createPrinter();
+  console.log("AST", ast)
+  const final = printer.printFile(ast)
```

Comment:
Please, create som automated jest tests. I would like to see different locale file code snippets with different formatting and assertion that we preserve formatting.